packages/lib/src/link/Link.tsx

```diff
@@ -107,8 +107,19 @@ const StyledLink = styled.div<{
        cursor: pointer;`}
   }
   &:focus {
-    border-radius: 2px;
-    outline: 2px solid ${(props) => props.theme.focusColor};
+    position: relative;
+    &::before {
+      content: "";
+      margin: -2px;
+      position: absolute;
+      top: 0;
+      right: 0;
+      bottom: 0;
+      left: 0;
+      border-radius: 4px;
+      outline: 2px solid ${(props) => props.theme.focusColor};
+      pointer-events: none;
+    }
```

Comment:
This is not the cleanest workaround but I haven't found anything else that worked as expected. Feel free to suggest a different solution.